tests/oss-fuzz/pdu_parse_target.c

```diff
@@ -2,12 +2,19 @@
 
 int
 LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
+
     coap_pdu_t *pdu = coap_pdu_init(0, 0, 0, size);
     if (pdu) {
         coap_set_log_level(LOG_DEBUG);
         coap_pdu_parse(COAP_PROTO_UDP, data, size, pdu);
+        coap_pdu_encode_header(pdu, COAP_PROTO_UDP);
```

Comment:
`coap_pdu_encode_header()` is for output and could possibly destroy the already parsed header structure in `pdu`.

Reply:
So, calling `coap_pdu_encode_header()` after `get_query()` and `get_uri_path()` would be better?

Reply:
You could call `coap_pdu_encode_header()` after `coap_pdu_show()` in this example. This would not make sense from application logic but would increase coverage.

Reply:
Alright I've moved it down after `coap_pdu_show()`